package.json

```diff
@@ -55,6 +55,7 @@
     "eslint-plugin-vue": "^5.0.0",
     "jest-transform-stub": "^2.0.0",
     "push-dir": "^0.4.1",
+    "lodash": "^4.17.11",
```

Comment:
I forget how this package changed its deployment, but I think it might be better to install the individual components of lodash you're using instead of the full library. EX: lodash.isempty above in the dependencies